.github/workflows/rundoc-ci.yml

```diff
@@ -0,0 +1,31 @@
+name: rundoc-ci-test
+on:
+  pull_request:
+    branches:
+      - main
+jobs:
+  test-rundoc:
+    runs-on: pub-hk-ubuntu-24.04-ip
+    steps:
+      - name: Get token for GH application (Linguist)
```

Comment:
We should be able to do the git checkout on this repo without a token, right? If so, drop this step.